test/gitlog.txt

```diff
@@ -24,12 +24,21 @@ Date:   Thu Sep 16 17:24:48 2021 +0200
 
     Temporal vocabulary (#146)
 
-commit 122dff83ef4f139e3361e5af557519c898ee81bd
+commit 122dff93ef4f139e3361e5af557519c898ee81bd
 Author: Ralf Handl <ralf.handl@sap.com>
 Date:   Thu Sep 16 17:19:01 2021 +0200
 
      Temporal: example for timeline with object key (#154)
     
-    CHANGELOG
+    CHANGELOG 122dff73ef4f139e3361e5af557519c898ee81bd
```

Comment:
I think we need a short handbook/README.md explaining this magic

Reply:
See https://github.com/oasis-tcs/odata-vocabularies/pull/156#issue-740479921

Reply:
Will someone find this pull request if they wonder about the magic?

Better add a section to `lib/README.md` that describes this cool mechanism.